en/upgrades/nonembedded-codelist-changelog.rst

```diff
@@ -342,7 +342,12 @@ Updates to other non-embedded codelists
      - Change description
      - Notes
      - Discussion
-   * - 20th August 2018
+   * - 6th December 2018
+     - :doc:`Currency </codelists/Currency>`
+     - Updated the Currency codelist to reflect changes to the ISO Currency codelist as of 6th December 2018.
+     - Added status 'withdrawn' to the VEF - Bolivar currency and added new code VES - Bolivar Soberano, as per the ISO currency codelist.
+     - See the discussion `here <https://discuss.iatistandard.org/t/update-currency-codelist-to-show-ves-bolivar-soberano/1591>`__      
+  * - 20th August 2018
```

Comment:
@amy-silcock @samuele-mattiuzzo There’s a syntax error here.

A space is missing at the start of this line, which means [the table no longer renders](https://github.com/IATI/IATI-Guidance/blob/89e7d24cc3ce8745aa602057c81d0a91e3fa33a0/en/upgrades/nonembedded-codelist-changelog.rst#updates-to-other-non-embedded-codelists).